js/adapt-contrib-scoring.js

```diff
@@ -34,8 +34,8 @@ export {
 class Scoring extends Backbone.Controller {
 
   initialize() {
-    this._rawSets = [];
     this.listenTo(Data, {
+      'loading': this.onDataLoading,
```

Comment:
Minor: Quotes can be removed
```suggestion
      loading: this.onDataLoading,
```